.github/renovate.json5

```diff
@@ -13,10 +13,10 @@
     suppressNotifications: ["prIgnoreNotification"],
     rebaseWhen: "conflicted",
     reviewers: ["team:modules"],
-    "pre-commit": {
     assigneesFromCodeOwners: true,
```

Comment:
Also opinions on these two things?

Ideally I'd like the workflow to be:
1. Maintainers of that module(codeowners) get assigned
2. Codeowners update the conda packages/tests/etc.(This should get fairly automatic with #4077).
3 Modules team to get review requested once the tests pass just to make sure things didn't get forgotten and just another set of eyes.
4. Automerge once modules team approves.

Reply:
sounds reasonable.